README.md

```diff
@@ -12,9 +12,10 @@ analogy queries as well as evaluation on analogy datasets.
 
 ## Installation
 
-Installing `finalfusion-utils` requires a Rust toolchain
-with minimum version `1.32` which can be installed via
-rustup.
+### Using `cargo`
+
+Installing `finalfusion-utils` requires a Rust toolchain with minimum
+version `1.40` which can be installed via rustup.
```

Comment:
Judging by CI, 1.37 should be the min version here

Reply:
Seems so! Somehow I misremembered that something needs 1.40.

Reply:
Fixed.